test/unit/services/shipwire_test.rb

```diff
@@ -177,29 +177,30 @@ def test_invalid_credentials
 
   def test_affiliate_id
     ActiveFulfillment::ShipwireService.affiliate_id = 'affiliate_id'
-
-    xml = REXML::Document.new(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
-    affiliate_id = REXML::XPath.first(xml, "//AffiliateId")
+    xml = Nokogiri::XML(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
+    affiliate_id = xml.at_xpath(xml, "//AffiliateId")
     assert_equal 'affiliate_id', affiliate_id.text
   end
 
   def test_company_name_in_request
-    xml = REXML::Document.new(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
-    company_node = REXML::XPath.first(xml, "//Company")
+    xml = Nokogiri::XML(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
+    company_node = xml.at_xpath(xml, "//Company")
     assert_equal 'MyCorp', company_node.text
   end
 
   def test_order_excludes_note_by_default
-    xml = REXML::Document.new(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
-    note_node = REXML::XPath.first(xml, "//Note").cdatas.first
-    assert_nil note_node
+    xml = Nokogiri::XML(@shipwire.send(:build_fulfillment_request, '123456', @address, @line_items, @options))
+    note_node = xml.at_xpath(xml, "//Note")
+    assert_equal note_node.children.select { |child| child.cdata? }.size, 0
+    assert_equal note_node.cdata?, false
```

Comment:
these asserts are backwards - the expected value should be first. Also for the second one it can just be `refute`